src/routes/Manual.svelte

```diff
@@ -108,107 +76,30 @@ $: hfDutyCycle = 50;
 $: single_pulse_duration = 500; //low priority
 
 
-$: single_pulse_block = [0,0,0,0];
+$: $single_pulse_block = [0,0,0,0];
 
 $: hfOn = hfPeriod * (hfDutyCycle/100);
-$: hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
+$: $hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
 
 $: lfOn = lfPeriod * (lfDutyCycle/100);
-$: lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
+$: $lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
 
 function setTimingBlock(config) {
     if (config == "infer") {
         hfPeriod = 10;      
-        lfPeriod = 0xff;
+        lfPeriod = 0xffff;
```

Comment:
What does this accomplish?

Reply:
It sets the command for the lf block to 255,255,255,255 instead of 255,0,255,0. Not sure how much that matters at this point but I figured I'd keep it consistent with the previous react version for now.